src/css/custom.css

```diff
@@ -5,6 +5,8 @@
  * work well for content-centric websites.
  */
 
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```

Comment:
per https://github.com/helium/docs/pull/1108, this should be removed and in the head instead

Reply:
the joys of github squishing... fixed!